.prettierrc.js

```diff
@@ -18,6 +18,7 @@ export default {
     '',
     '^@oxide/(.*)$',
     '',
+    '^~/(.*)$',
     '^app/(.*)$',
```

Comment:
This is about sorting `~/` into the right spot when prettier sorts and groups imports.